apps/librelingo_json_export/librelingo_json_export/challenge_types.py

```diff
@@ -92,6 +92,27 @@ def get_options_challenge(phrase, _):
         }
     ]
 
+def get_pairing_challenge(words, _):
+    if not isinstance(words, list):
```

Comment:
I realized that this part is odd. This should not be needed whatsoever :thinking: Instead, the tests should always operate on a set of words to begin with, therefore it should always be called with a list of words

Reply:
This could be partially the reason for the errors that we are seeing, because this magic here might be creating some confusion that makes it harder to debug what's happening there